aws/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "access_key" {
```

Comment:
What's the point of this file really?

Reply:
At first I had all my secrets in main.tf. I had main.tf in .gitignore, then saw the terraform docs breaking these up into individual files. Idkwtf im doing. Trying to figure it out though.